.github/workflows/ci.yml

```diff
@@ -0,0 +1,56 @@
+name: GitHub CI
+
+on:
+  push:
+  pull_request:
+    paths-ignore:
+      - 'README.md'
```

Comment:
```suggestion
      - '*.md'
      - '.github/*'
      - '.github/ISSUE_TEMPLATE/**'
```

Reply:
I think it's better to add `'.github/ISSUE_TEMPLATE/**'` to push's paths-ignore as well.

Reply:
> It should work for `--build`, otherwise I have no idea.

Weird, I retried it again. Now it's working. Except it doesn't work with other platforms.